community/resources/resources.md

```diff
@@ -105,62 +39,6 @@ We have [channels][] for each project, but the following channels are recommende
 [appdesign]: https://magentocommeng.slack.com/messages/CBSL1DF8B
 [coding-standards]: https://magentocommeng.slack.com/messages/CFC88F1C6
 [announcements]: https://magentocommeng.slack.com/messages/C7FA71S3V
-[0]: https://github.com/DavidLambauer/awesome-magento2
+[0]: https://magento.com/blog
```

Comment:
Shouldn't this be the DevBlog? If not, we definitely need a link to it in addition to the Marketing blog.